src/components/Storyblok/SbStoryCard.tsx

```diff
@@ -2,61 +2,72 @@ import React from 'react';
 import { storyblokEditable } from 'gatsby-source-storyblok';
 import { AnimationType } from '../Animate';
 import { HeadingType } from '../Typography';
-import { VerticalCard } from '../VerticalCard';
+import { StoryCard } from '../VerticalCard';
 import { SbImageType, SbLinkType } from './Storyblok.types';
 import { AccentBgColorType } from '../../utilities/datasource';
 import { paletteAccentColors, PaletteAccentColorType } from '../../utilities/colorPalettePlugin';
 
-export type SbVerticalCardProps = {
+export type SbStoryCardProps = {
   blok: {
     _uid: string;
+    storyPicker?: {
+      content?: {
+        title?: string;
+        cardTitle?: string;
+        topics?: string[];
```

Comment:
Here we're getting "topics" which is a taxonomy field (multiple-options checkboxes in Storyblok). Passing this array of strings (the checked values) into the Story Card component to be rendered into tag links